<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bookmark Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitterFT">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="BookmarkFoldersView" name="tf" native="true"/>
       <widget class="TagsView" name="tv" native="true"/>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
          <item>
           <widget class="QLabel" name="lblFilter">
            <property name="styleSheet">
             <string notr="true">QLabel { font-weight: bold; }</string>
            </property>
            <property name="text">
             <string>Showing All Bookmarks</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblBMCount">
            <property name="styleSheet">
             <string notr="true">QLabel { font-weight: bold; }</string>
            </property>
            <property name="text">
             <string>0 Bookmarks</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lblSearch">
            <property name="text">
             <string>&amp;Show bookmarks containing:</string>
            </property>
            <property name="buddy">
             <cstring>leSearch</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSearch"/>
          </item>
          <item>
           <widget class="QCheckBox" name="chkSearchRegExp">
            <property name="text">
             <string>RegExp</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="BookmarksView" name="bv" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnNew">
            <property name="text">
             <string>&amp;New</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnView">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;View</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelete">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_importFirefoxBookmarks">
   <property name="text">
    <string>Import Firefox Bookmarks...</string>
   </property>
  </action>
  <action name="actionImportFirefoxBookmarksJSONfile">
   <property name="text">
    <string>Import Firefox Bookmarks JSON File...</string>
   </property>
  </action>
  <action name="actionGetMHT">
   <property name="text">
    <string>GetMHT</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionImportUrlsAsBookmarks">
   <property name="text">
    <string>Import URL(s) as Bookmarks...</string>
   </property>
   <property name="toolTip">
    <string>Import one or more URLs as Bookmarks</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BookmarksView</class>
   <extends>QWidget</extends>
   <header>Bookmarks/BookmarksView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BookmarkFoldersView</class>
   <extends>QWidget</extends>
   <header>BookmarkFolders/BookmarkFoldersView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TagsView</class>
   <extends>QWidget</extends>
   <header>Tags/TagsView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnNew</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
